<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RangeSelector</class>
 <widget class="QWidget" name="RangeSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButtonApply">
     <property name="text">
      <string>Aplicar</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QDial" name="dialCenterSelector">
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>4095</number>
         </property>
         <property name="pageStep">
          <number>2000</number>
         </property>
         <property name="sliderPosition">
          <number>128</number>
         </property>
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="notchTarget">
          <double>16.699999999999999</double>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>18</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_c">
           <property name="text">
            <string>Centro</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>4096</number>
           </property>
           <property name="value">
            <number>128</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>18</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QDial" name="dialWidthSelector">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>2048</number>
         </property>
         <property name="pageStep">
          <number>1000</number>
         </property>
         <property name="value">
          <number>128</number>
         </property>
         <property name="sliderPosition">
          <number>128</number>
         </property>
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="notchTarget">
          <double>14.699999999999999</double>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>17</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_r">
           <property name="text">
            <string>Rango</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_2">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>2048</number>
           </property>
           <property name="value">
            <number>128</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>18</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dialCenterSelector</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialWidthSelector</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dialCenterSelector</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dialWidthSelector</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>dialCenterSelector</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>editingFinished()</signal>
   <receiver>dialWidthSelector</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
